.gitattributes

```
@@ -36,9 +36,9 @@
 /demos/** omz.ci.job-for-change.demos
 /demos/**/*.md -omz.ci.job-for-change.demos
 
-/models/**/*.md omz.ci.job-for-change.documentation
-
 /tools/accuracy_checker/** omz.ci.job-for-change.ac
 /tools/accuracy_checker/configs/*.yml -omz.ci.job-for-change.ac
 /tools/downloader/** omz.ci.job-for-change.downloader
 /tools/**/*.md -omz.ci.job-for-change.ac -omz.ci.job-for-change.downloader
+
+**/*.md omz.ci.job-for-change.documentation
```

```suggestion
*.md omz.ci.job-for-change.documentation
```

`**/` in front is redundant.

I also suggest to merge this rule with the other rules for Markdown files (i.e., disable all non-documentation jobs for changes to `*.md`).

---

Done.

---

No you haven't. You need to disable the other jobs for changes to `*.md`.

---

Oops, I didn't notice "-" sign; fixed.